Better describe creating the chef-repo

tas50 opened the pull request:
Better describe the command to run and link to the docs for that command.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Better describe creating the Chef Repo

Better describe the command to run and link to the docs for that command.

Signed-off-by: Tim Smith <tsmith@chef.io>

kagarmoe commented:
Could you expand a bit about _why_ a user needs a chef repo up in the first paragraphs? I've never felt that it was clear.

tas50 commented:
Yeah this whole thing is pretty terrible really. Let me at least do the elevator pitch

tas50 committed:
Rework the whole chef-repo page to be about Policyfiles

We need to document our desired behavior for new users here. This whole page is for brand new users and we don't want them starting out with roles/environments.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Expand the first paragraph

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Replicate a dtag

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 commented:
@kagarmoe How does this look now?

tas50 committed:
Update the directory structure section as well

Signed-off-by: Tim Smith <tsmith@chef.io>

mjingle commented on chef_master/source/auth.rst:
```diff
@@ -81,16 +81,15 @@ Each workstation stores its private key in the chef-repo. This private key is ge
 .. tag chef_repo_description
 
-The chef-repo is a directory on your workstation that stores:
+The chef-repo is a directory on your workstation that stores everything you need to define you infrastructure with Chef Infra:
```
```suggestion
The chef-repo is a directory on your workstation that stores everything you need to define your infrastructure with Chef Infra:
```

tas50 committed:
Update chef_master/source/auth.rst

Signed-off-by: Tim Smith <tsmith@chef.io>

Co-Authored-By: mjingle <mjinglewski@chef.io>

tas50 committed:
Replicate the dtag

Signed-off-by: Tim Smith <tsmith@chef.io>

kagarmoe commented on chef_master/source/auth.rst:
```diff
@@ -81,16 +81,15 @@ Each workstation stores its private key in the chef-repo. This private key is ge
 The chef-repo directory should be synchronized with a version control system, such as git. All of the data in the chef-repo should be treated like source code.
 
-knife is used to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, that data is used by Chef Infra Client to manage all of the nodes that are registered with the Chef Infra Server and to ensure that the correct cookbooks, environments, roles, and other settings are applied to nodes correctly.
+The chef and knife commands are used to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, that data is used by Chef Infra Client to manage all of the nodes that are registered with the Chef Infra Server and to ensure that the correct cookbooks, policyfiles, and other settings are applied to nodes correctly.
```
```suggestion
You'll use the ``chef`` and ``knife ``commands to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, Chef Infra Client uses that data to manage the nodes registered with the Chef Infra Server and to ensure that it applies the right cookbooks, policyfiles, and settings to the right nodes in the right order.
```

Does the repetition of "right" work for you? Is it clarifying or distracting? (I'm on the fence)

tas50 replied:
double right seems kinda odd

kagarmoe replied:
I was thinking about doing something in the right place, at the right time, in the right order. Rhetorically sound, but perhaps the wrong audience.

Commit:
Merge branch 'master' into repo_setup

tas50 committed:
Update chef_master/source/auth.rst

Signed-off-by: Tim Smith <tsmith@chef.io>

Co-Authored-By: Kimberly Garmoe <kgarmoe@chef.io>

IanMadd commented on chef_master/source/auth.rst:
```diff
@@ -89,7 +89,7 @@ The chef-repo is a directory on your workstation that stores everything you need
 The chef-repo directory should be synchronized with a version control system, such as git. All of the data in the chef-repo should be treated like source code.
 
-The chef and knife commands are used to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, that data is used by Chef Infra Client to manage all of the nodes that are registered with the Chef Infra Server and to ensure that the correct cookbooks, policyfiles, and other settings are applied to nodes correctly.
+You'll use the ``chef`` and ``knife ``commands to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, Chef Infra Client uses that data to manage the nodes registered with the Chef Infra Server and to ensure that it applies the right cookbooks, policyfiles, and settings to the right nodes in the right order.
```
I think this will fix the Sphinx error.
```suggestion
You'll use the ``chef`` and ``knife`` commands to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, Chef Infra Client uses that data to manage the nodes registered with the Chef Infra Server and to ensure that it applies the right cookbooks, policyfiles, and settings to the right nodes in the right order.
```

tas50 committed:
Update chef_master/source/auth.rst

Signed-off-by: Tim Smith <tsmith@chef.io>

Co-Authored-By: Ian Maddaus <IanMadd@users.noreply.github.com>

Commit:
Fix dtag

Signed-off-by: IanMadd <imaddaus@chef.io>

IanMadd committed:
Merge branch 'master' into repo_setup

IanMadd reviewed (approved):
![tenor-202262284](https://user-images.githubusercontent.com/8138169/73992364-c4943c00-4903-11ea-8a1a-d0a38cf305de.gif)
